modules/oe_theme_content_entity_contact/oe_theme_content_entity_contact.post_update.php

```diff
@@ -34,3 +35,33 @@ function oe_theme_content_entity_contact_post_update_00001(): void {
     }
   }
 }
+
+/**
+ * Create the oe_contact 'full' view mode.
+ */
+function oe_theme_content_entity_contact_post_update_00002(): void {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_theme_content_entity_contact') . '/config/post_updates/00002_create_full_view_display');
+
+  $config_manager = \Drupal::service('config.manager');
+  $entity_type_manager = \Drupal::entityTypeManager();
+
+  // View display configurations to update.
+  $names = [
+    'core.entity_view_mode.oe_contact.full',
+    'core.entity_view_display.oe_contact.oe_general.full',
+    'core.entity_view_display.oe_contact.oe_press.full',
+  ];
+  foreach ($names as $name) {
+    $config = $storage->read($name);
+    // We are creating the config which means that we are also shipping
+    // it in the config/install folder so we want to make sure it gets the hash
+    // so Drupal treats it as a shipped config. This means that it gets exposed
+    // to be translated via the locale system as well.
+    $config['_core']['default_config_hash'] = Crypt::hashBase64(serialize($config));
```

Comment:
Since this comes from a post update directory, can't we just add it to the YAML itself?